### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #811 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#412]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
Started by GitHub push by KaplanAlexander
Building in workspace /var/lib/jenkins/sharedspace/SmartCity-Accessibility
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/TechnionYP5777/SmartCity-Accessibility.git # timeout=10
Fetching upstream changes from https://github.com/TechnionYP5777/SmartCity-Accessibility.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/TechnionYP5777/SmartCity-Accessibility.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 990cade6052834ca3e57f4abcc49b6436e2fe0db (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 990cade6052834ca3e57f4abcc49b6436e2fe0db
 > git rev-list d9584735e81a58889752d470848251289615d06b # timeout=10
Injecting SonarQube environment variables using the configuration: Sonar
[SmartCity-Accessibility] $ /var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/Maven/bin/mvn -f server-side/pom.xml clean -Dram=1224 org.jacoco:jacoco-maven-plugin:prepare-agent test -Dsonar.coverage.exclusions=**gui** -Dgroups=smartcity.accessibility.categories.UnitTests
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building SmartCity-Accessibility 0.5.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] e3620767a963c48ba3948b749dd0e6cbe47467c3 - jumping into deep water (removing jxmaps immediately) #409

- [Alexander Kaplan] 8f571a5cf6781c8499ffb1830d76125039f01f19 - replacing LatLng #409

- [Alexander Kaplan] 7033bcf512a7573e7c71b25668b1a8a219c6ac92 - added gmaps api dep #409

- [Alexander Kaplan] 5b83ff8cb9d448d33da8c570775cd67561ddfbcf - Search package done #409

- [Alexander Kaplan] 1409137c66c3809b26438aa5a3ccb66a84442653 - services done #409

- [Alexander Kaplan] 7ab07a614999bf2c2eaac6fcf8b654d1ee2b1e37 - tests done #409

- [Alexander Kaplan] db8807ac68f6328af9ce433829a8b994d14058d4 - all tests now pass .... that was quick #409

- [Alexander Kaplan] 0ba20b8f432aeafd8803ce04714a215b6c39aecc - remove warnings and logger test print #409

- [Alexander Kaplan] 3325ae0f5a9455a721840771f1bb9fe16457be6a - done subtype conversion #409

- [Alexander Kaplan] 0c436b7061881479cd3a62d4a123e43a995b2a9c - test reworked to fit #409

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/811/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
Started by user yaelAmitay
Building in workspace /var/lib/jenkins/sharedspace/SmartCity-Accessibility
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/TechnionYP5777/SmartCity-Accessibility.git # timeout=10
Fetching upstream changes from https://github.com/TechnionYP5777/SmartCity-Accessibility.git
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/TechnionYP5777/SmartCity-Accessibility.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/master^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/master^{commit} # timeout=10
Checking out Revision 990cade6052834ca3e57f4abcc49b6436e2fe0db (refs/remotes/origin/master)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 990cade6052834ca3e57f4abcc49b6436e2fe0db
 > git rev-list 990cade6052834ca3e57f4abcc49b6436e2fe0db # timeout=10
Injecting SonarQube environment variables using the configuration: Sonar
[SmartCity-Accessibility] $ /var/lib/jenkins/tools/hudson.tasks.Maven_MavenInstallation/Maven/bin/mvn -f server-side/pom.xml clean -Dram=1224 org.jacoco:jacoco-maven-plugin:prepare-agent test -Dsonar.coverage.exclusions=**gui** -Dgroups=smartcity.accessibility.categories.UnitTests
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building SmartCity-Accessibility 0.5.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
Downloading: https://repo.maven.apache.org/maven2/com/nimbusds/lang-tag/maven-metadata.xml
Downloading: https://jitpack.io/com/nimbusds/lang-tag/maven-metadata.xml
Downloading: http://repo.spring.io/libs-release/com/nimbusds/lang-tag/maven-metadata.xml
Downloading: http://kapalex.ddns.net:8081/artifactory/smartcity-accessibility/com/nimbusds/lang-tag/maven-metadata.xml
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] e3620767a963c48ba3948b749dd0e6cbe47467c3 - jumping into deep water (removing jxmaps immediately) #409

- [Alexander Kaplan] 8f571a5cf6781c8499ffb1830d76125039f01f19 - replacing LatLng #409

- [Alexander Kaplan] 7033bcf512a7573e7c71b25668b1a8a219c6ac92 - added gmaps api dep #409

- [Alexander Kaplan] 5b83ff8cb9d448d33da8c570775cd67561ddfbcf - Search package done #409

- [Alexander Kaplan] 1409137c66c3809b26438aa5a3ccb66a84442653 - services done #409

- [Alexander Kaplan] 7ab07a614999bf2c2eaac6fcf8b654d1ee2b1e37 - tests done #409

- [Alexander Kaplan] db8807ac68f6328af9ce433829a8b994d14058d4 - all tests now pass .... that was quick #409

- [Alexander Kaplan] 0ba20b8f432aeafd8803ce04714a215b6c39aecc - remove warnings and logger test print #409

- [Alexander Kaplan] 3325ae0f5a9455a721840771f1bb9fe16457be6a - done subtype conversion #409

- [Alexander Kaplan] 0c436b7061881479cd3a62d4a123e43a995b2a9c - test reworked to fit #409

No changes


[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/812/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
